wolfssl/wolfcrypt/port/caam/wolfcaam.h

```diff
@@ -39,25 +46,107 @@ WOLFSSL_LOCAL int wc_caamFreeRng(void);
 
 WOLFSSL_LOCAL word32 wc_caamReadRegister(word32 reg);
 WOLFSSL_LOCAL void wc_caamWriteRegister(word32 reg, word32 value);
-WOLFSSL_LOCAL int  wc_caamAddAndWait(Buffer* buf, word32 arg[4], word32 type);
+WOLFSSL_LOCAL int  wc_caamAddAndWait(CAAM_BUFFER* buf, int sz, word32 arg[4],
```

Comment:
Should `arg` should be `word32* arg`. Now that `sz` is a parameter. Also there is not actual checking or use of the size when looking at the code.

Reply:
sz here is the number of CAAM_BUFFER's